Could not resolve "zipcelx"

## Describe the bug

If I upgrade to version `19.9.0` or higher, I get the build error that `zipcelx` is not found.

```
[INFO] ✘ [ERROR] Could not resolve "zipcelx"
[INFO] 
[INFO]     node_modules/.pnpm/@porscheinformatik+clr-addons@19.9.0_@angular+common@20.1.7_@angular+core@20.1.7_@angul_6cc2b783dff2af672d05f6a08b57587f/node_modules/@porscheinformatik/clr-addons/fesm2022/clr-addons.mjs:19:20:
[INFO]       19 │ import zipcelx from 'zipcelx';
[INFO]          ╵                     ~~~~~~~~~
[INFO] 
[INFO]   You can mark the path "zipcelx" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
```

## To Reproduce

Don't have `zipcelx` as dependency in your project and increase version to `19.9.0` or higher.

## Expected behavior

No build error.

## Versions

**App**

- Angular: 20.1.5/6
- Node: 22.17.0
- Clarity: 17.10.0
- Clarity-Addons: 19.8.0 -> 19.9.0

## Additional context

Caused by #2645; slightly related to #1976

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "19.9.1",
+  "version": "19.9.2",
   "private": true,
   "scripts": {
     "build:ui:css": "sass --source-map --precision=8 ./src/clr-addons/themes/phs/phs-theme.scss ./dist/clr-addons/styles/clr-addons-phs.css",
@@ -53,8 +53,15 @@
     "core-js": "3.43.0",
     "rxjs": "7.8.2",
     "tslib": "2.8.1",
-    "zipcelx": "1.6.2",
     "zone.js": "0.15.0"
+  },
+  "peerDependencies": {
+    "zipcelx": "1.6.2"
+  },
+  "peerDependenciesMeta": {
+    "zipcelx": {
+      "optional": true
+    }
   },
   "devDependencies": {
     "@alasdair/karma-scss-preprocessor": "5.0.0-3",

--- a/src/clr-addons/clr-addons.module.ts
+++ b/src/clr-addons/clr-addons.module.ts
@@ -42,7 +42,6 @@ import { ClrDaterangepickerModule } from './daterangepicker';
 import { ClrIfWarningModule } from './clr-control-warning/if-warning.module';
 import { ClrActionPanelModule } from './action-panel';
 import { ClrReadonlyDirectiveModule } from './readonly/readonly.module';
-import { ClrExportDatagridButtonModule } from './export-datagrid-button/export-datagrid-button.module';
 
 @NgModule({
   exports: [
@@ -80,7 +79,6 @@ import { ClrExportDatagridButtonModule } from './export-datagrid-button/export-d
     ClrIfWarningModule,
     ClrActionPanelModule,
     ClrReadonlyDirectiveModule,
-    ClrExportDatagridButtonModule,
     ClrDatagridColumnReorderModule,
   ],
 })

--- a/src/clr-addons/export-datagrid-button/export-datagrid.service.ts
+++ b/src/clr-addons/export-datagrid-button/export-datagrid.service.ts
@@ -1,10 +1,22 @@
-import zipcelx, { ZipCelXConfig } from 'zipcelx';
+import { ZipCelXConfig } from 'zipcelx';
 import { Injectable } from '@angular/core';
 
-@Injectable({
-  providedIn: 'root',
-})
+@Injectable()
 export class ExportDatagridService {
+  zipcelx: any;
+
+  constructor() {
+    this.init();
+  }
+
+  async init() {
+    try {
+      this.zipcelx = await import('zipcelx');
+    } catch (error) {
+      console.warn('Optional feature requires `zipcelx`. Please install it: npm install zipcelx', error);
+    }
+  }
+
   exportToExcel(
     filename: string,
     headers: { value: string; type: string }[],
@@ -16,6 +28,7 @@ export class ExportDatagridService {
         data: [headers, ...rows],
       },
     };
-    zipcelx(config);
+
+    this.zipcelx.default(config);
   }
 }

--- a/src/clr-addons/package.json
+++ b/src/clr-addons/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@porscheinformatik/clr-addons",
-  "version": "19.9.1",
+  "version": "19.9.2",
   "description": "Addon components for Clarity Angular",
   "es2015": "esm2015/clr-addons.js",
   "homepage": "https://porscheinformatik.github.io/clarity-addons/",

--- a/src/dev/src/app/exportable-datagrid/exportable-datagrid.module.ts
+++ b/src/dev/src/app/exportable-datagrid/exportable-datagrid.module.ts
@@ -3,7 +3,7 @@ import { ExportableDatagridComponent } from './exportable-datagrid.component';
 import { RouterModule } from '@angular/router';
 import { ClrDatagridModule } from '@clr/angular';
 import { CommonModule } from '@angular/common';
-import { ClrAddonsModule } from '@porscheinformatik/clr-addons';
+import { ClrAddonsModule, ClrExportDatagridButtonModule } from '@porscheinformatik/clr-addons';
 
 @NgModule({
   imports: [
@@ -11,6 +11,7 @@ import { ClrAddonsModule } from '@porscheinformatik/clr-addons';
     RouterModule.forChild([{ path: '', component: ExportableDatagridComponent }]),
     ClrDatagridModule,
     ClrAddonsModule,
+    ClrExportDatagridButtonModule,
   ],
   declarations: [ExportableDatagridComponent],
   exports: [ExportableDatagridComponent],

--- a/website/src/app/documentation/demos/datagrid/datagrid.demo.module.ts
+++ b/website/src/app/documentation/demos/datagrid/datagrid.demo.module.ts
@@ -11,7 +11,7 @@ import { DatagridDemo } from './datagrid.demo';
 import { DocWrapperModule } from '../_doc-wrapper/doc-wrapper.module';
 import { RouterModule } from '@angular/router';
 import { UtilsModule } from '../../../utils/utils.module';
-import { ClrAddonsModule } from '@porscheinformatik/clr-addons';
+import { ClrAddonsModule, ExportDatagridButtonComponent } from '@porscheinformatik/clr-addons';
 import { CdkDrag, CdkDropList } from '@angular/cdk/drag-drop';
 
 @NgModule({
@@ -19,6 +19,7 @@ import { CdkDrag, CdkDropList } from '@angular/cdk/drag-drop';
     CommonModule,
     ClarityModule,
     ClrAddonsModule,
+    ExportDatagridButtonComponent,
     DocWrapperModule,
     UtilsModule,
     RouterModule.forChild([{ path: '', component: DatagridDemo }]),

